docker/.dockerignore

```diff
@@ -3,6 +3,9 @@
 !/domains
 !/shared
 !/test
+!/bump-versions.sh
 !/Cargo.lock
 !/Cargo.toml
+!/runtime-benchmark.sh
 !/rust-toolchain.toml
+!/rustfmt.toml
```

Comment:
Should we also be excluding `.git`, `.github`, and `.gitignore` in Docker?

Reply:
This is not an exclude file (not `!` at the beginning of each file). We exclude everything (see the first line) and then have a whitelist of things to actually include. Are you sure the new entries are actually needed when building a container image? I don't think so.

Whitelist ensures no random files are included by accident and any missing files cause build errors, ensuring you have just the things needed. Nothing more, nothing less.